doc/source/generation.rst

```diff
@@ -7,6 +7,10 @@ The :class:`Graph` class is the main object used to generate graphs:
 
 >>> from igraph import Graph
 
+To copy a graph, use :meth:`Graph.copy`:
```

Comment:
Mention that this is a "shallow" copy, i.e. the structure of the graph is copied, but any mutable objects in the attributes are _not_ copied (they would refer to the same instance). Also mention that if someone really needs a deep copy, they should use the `deepcopy` module from Python, which basically does a pickling - unpickling sequence to achieve a deep copy.

Reply:
yeah added